ChangeLog

```diff
@@ -81,6 +81,10 @@ What's New in Pylint 1.8?
       c-extensions.
       Close #1466
 
+    * Fix a false positive ``useless-super-delegation`` message when
```

Comment:
This will also need a What's new Entry, in the ``Other Changes`` section. Also please add yourself to Contributors file if you are not already there.